examples/misc/logos2.py

```diff
@@ -1,87 +1,170 @@
 """
-================
-Matplotlib Logos
-================
+===============
+Matplotlib logo
+===============
 
-Displays some matplotlib logos.
+This example generates the current matplotlib logo.
 
-Thanks to Tony Yu <tsyu80@gmail.com> for the logo design
+Thanks to Tony Yu <tsyu80@gmail.com> for the original logo design.
```

Comment:
Does anybody know who is responsible for the current logo?

Reply:
The log for the file goes back to 4d11bf46002f91b846899059c778abab3e218b1b, so JDH? There's not code there either, so hard to say. Possibly it came from some separate website repo...

Reply:
Nope, sorry, that's the old logo. The new one was added in #5653 and was apparently the logo shipped as stickers for a year before that. I don't know if it was @mdboom who designed it or someone else.

Reply:
I have moved the original logo credit to the credits page. It's better to collect all credits there and not scatter them throughout the documentation. If anybody wants to add credit for the current logo, please do there.